group_membership.tf

```diff
@@ -2,7 +2,7 @@
 resource "aws_iam_user_group_membership" "provisioner" {
   provider = aws.users
 
-  for_each = { for username, attributes in var.users : username => "" if contains(attributes["roles"], "provisioner") }
+  for_each = toset([for username, attributes in var.users : username if contains(attributes["roles"], "provisioner")])
```

Comment:
Hallelujah!

Reply:
Ha- there's one place to make this change, over in [`cool-auditor-iam`](https://github.com/cisagov/cool-auditor-iam/blob/develop/group_membership.tf) - I'll get to it soon.